test/Polly.Specs/Polly.Specs.csproj

```diff
@@ -27,6 +27,7 @@
     <Using Include="Xunit" />
     <Using Include="Xunit.Abstractions" />
     <Using Include="Xunit.Sdk" />
+    <Using Include="Polly.Specs.DictionaryHelpers" Static="true" />
```

Comment:
Could you put this in the appropriate place in the sort please?

Reply:
Are they in sorted order by name?

Reply:
Yes - if you look at the `ItemGroup` you see that they go `F => N => P => X` in alphabetical order by the value of `Include`.

Reply:
Got it thanks